apps/channels/serializers.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework import serializers
+
+
+class MessageSerializer(serializers.Serializer):
+    timestamp = serializers.CharField()
+    message_id = serializers.UUIDField()
+    ciphertext = serializers.CharField()
+    tag = serializers.CharField()
+    nonce = serializers.CharField()
+
+
+class CommCareConnectMessageSerializer(serializers.Serializer):
+    channel_id = serializers.UUIDField()
+    messages = serializers.ListField(child=MessageSerializer())
```

Comment:
The recommended way to do this is with `many=True`:
```suggestion
    messages = MessageSerializer(many=True)
```

See https://www.django-rest-framework.org/api-guide/serializers/#dealing-with-nested-objects

Reply:
Ah thanks: https://github.com/dimagi/open-chat-studio/pull/1041/commits/210136831ed9dad0af7614616ff616c166bc5314